server/middleware/cacher.js

```diff
@@ -0,0 +1,32 @@
+// This middleware checks for Pokemon Id's and if they are stored in "cache", removes them from the request.
+let cachedIDs = [];
+setInterval(() => {
+   cachedIDs = [];
+}, 60000);
+
+module.exports = function cacher(req, res, next) {
+   if (
+      req.body.task
+         .replace(/\s/g, "")
+         .split(",")
+         .every((elem) => !isNaN(elem))
+   ) {
+      let idsToPass = [];
+      const pokemonIDS = req.body.task.replace(/\s/g, "").split(","); // "1, 2, 3" => [1,2,3]
```

Comment:
you can reuse `pokemonIDS`
```suggestion
  const potentialPokemonIDS = req.body.task.replace(/\s/g, "").split(","); // "1, 2, 3" => [1,2,3]
   if (
      req.potentialPokemonIDS.every((elem) => !isNaN(elem))
   ) {
      const pokemonIDS = potentialPokemonIDS;
      let idsToPass = [];
```